lib/kafka-worker.rb

```diff
@@ -101,7 +120,9 @@ def handle(message)
     end
 
     def on_error(message, err)
-      logger.error("#{self.class.name} failed on message: #{message.inspect} with error: #{err}")
+      error_message = "#{self.class.name} failed on message: #{message.inspect} with error: #{err}"
+      logger.error(error_message)
+      Rollbar.error(error_message)
```

Comment:
also, better to call `Rollbar.error` with the exception, then we get more details in rollbar! The message can be attached to the payload we're sending to Rollbar I believe, check the docs.